docs/reference/modules/network.asciidoc

```diff
@@ -26,7 +26,7 @@ of these.
 +
 Defaults to `_local_`.
 
-`discovery.zen.ping.unicast.hosts`::
+`discovery.zen.pinger.unicast.hosts`::
```

Comment:
I suspect this was unintentional. I see replacements of `ping` to `pinger` throughout. It should be reverted though.